fuzz/wrap.c

```diff
@@ -85,9 +92,12 @@ extern int __wrap_asprintf(char **strp, const char *fmt, ...) {
     return -1;
   }
 
+#pragma clang diagnostic push
```

Comment:
Would annotating the wrapper function with the `format` attribute (see e.g. debug.h) work?